packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -262,17 +262,21 @@ contract Deploy is Deployer {
     /// @notice Deploy the DisputeGameFactoryProxy
     function deployDisputeGameFactoryProxy() onlyDevnet broadcast() public returns (address) {
         address proxyAdmin = mustGetAddress("ProxyAdmin");
-        Proxy proxy = new Proxy({
-            _admin: proxyAdmin
-        });
 
```

Comment:
Do we need both of these in the long term?

Reply:
Not necessarily, if we'd like to no longer test the alphabet game, we can remove it. We can also have different game type mappings for devnet - this was just to make it easy with `GameType.FAULT = 0`

Reply:
I would say we want to keep alphabet and cannon long term.  It is _way_ cheaper to test the game and challenger with alphabet than with cannon. Plus it's much easier to create understandable differences in an alphabet trace than a cannon trace.